app/src/main/java/com/marcelorcorrea/falae/activity/DisplayActivity.kt

```diff
@@ -3,24 +3,30 @@ package com.marcelorcorrea.falae.activity
 import android.content.Intent
 import android.os.Bundle
 import android.support.v7.app.AppCompatActivity
+import android.view.KeyEvent
 import android.widget.Toast
 import com.marcelorcorrea.falae.R
 import com.marcelorcorrea.falae.fragment.PageFragment
+import com.marcelorcorrea.falae.fragment.SettingsFragment
 import com.marcelorcorrea.falae.fragment.ViewPagerItemFragment
 import com.marcelorcorrea.falae.model.Page
 import com.marcelorcorrea.falae.model.SpreadSheet
 import com.marcelorcorrea.falae.service.TextToSpeechService
+import com.marcelorcorrea.falae.storage.SharedPreferencesUtils
 
 class DisplayActivity : AppCompatActivity(), PageFragment.PageFragmentListener, ViewPagerItemFragment.ViewPagerItemFragmentListener {
 
     private var currentSpreadSheet: SpreadSheet? = null
+    private var scanMode: Boolean = false
```

Comment:
This flag is leaking, shouldn't be here.
I think you could move this flag to ViewPagerItemFragment or even PageFragment and call a method that handles this flag and select the highlighted item.